nnf/bin/README.md

```diff
@@ -0,0 +1,5 @@
+# Binary files included with python-nnf
+
+## kissat
+
+The `kissat` SAT solver is released under MIT license, and can be found [here](https://github.com/arminbiere/kissat/). The binary included in this directory corresponds to the statically linked version at [this commit](https://github.com/arminbiere/kissat/commit/baef4609163f542dc08f43aef02ce8da0581a2b5). No modifications were made to the `kissat` solver.
```

Comment:
I think we'd have to include the license itself too, e.g. as a file `LICENSE.kissat`, based on ["The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software."](https://github.com/arminbiere/kissat/blob/master/LICENSE)